requirements.txt

```diff
@@ -1,26 +1,24 @@
-git+https://github.com/raiden-network/raiden.git@ff813c73b2f12e29e6d13281d61f1da0c274c4a8
+git+https://github.com/raiden-network/raiden.git@aa189fd32dd3df29d8c1a70f5c2bf5880273050e
 
-raiden-contracts==0.22.0
+raiden-contracts==0.23.0
 
-structlog==18.2.0
-colorama==0.3.9
+structlog==19.1.0
+colorama==0.4.1
 
 click==7.0
-coincurve==11.0.0
+coincurve==12.0.0
 
-web3==4.9.1
+web3==4.9.2
 eth-utils==1.4.1
 eth-abi==1.3.0
 eth-account==0.3.0
-py_ecc<=1.5.0
-py-solc==3.2.0
-rlp==1.1.0
+eth-typing==2.1.0
 
-flask==1.0.2
-flask_restful==0.3.6
-gevent==1.3.6
-networkx==2.1
-requests==2.20.0
+flask==1.0.3
+flask-restful==0.3.7
+gevent==1.3.7
+networkx==2.3
+requests==2.22.0
 
 marshmallow==3.0.0rc6
-marshmallow-dataclass==6.0.0c2
```

Comment:
Why didn't this cause an error?

Reply:
No clue, I wonder more and more about `pip`.